packages/ceramic-common/src/logger-provider.ts

```diff
@@ -1,8 +1,6 @@
 import chalk from 'chalk'
```

Comment:
Could chalk also be moved to the logger plugin please? AFAICT it only targets node.

Reply:
`chalk` dependency is not really necessary. @valmack feel free to remove it